test/scripts/scripts.test.js

```diff
@@ -0,0 +1,36 @@
+import { expect } from '@esm-bundle/chai';
+import { setLibs, LIBS } from '../../scripts/scripts.js';
+
+describe('Libs', () => {
+  const tests = [
+    ['https://business.adobe.com', '/libs'],
+    ['https://business.adobe.com?milolibs=foo', '/libs'],
+    ['https://business.stage.adobe.com', 'https://main--milo--adobecom.hlx.live/libs'],
+    ['https://business.stage.adobe.com?milolibs=foo', 'https://foo--milo--adobecom.hlx.live/libs'],
+    ['https://business.stage.adobe.com?milolibs=awesome--milo--forkedowner', 'https://awesome--milo--forkedowner.hlx.live/libs'],
+    ['https://main--bacom--adobecom.hlx.page/', 'https://main--milo--adobecom.hlx.live/libs'],
+    ['https://main--bacom--adobecom.hlx.page/?milolibs=foo', 'https://foo--milo--adobecom.hlx.live/libs'],
+    ['https://main--bacom--adobecom.hlx.page/?milolibs=local', 'http://localhost:6456/libs'],
+    ['https://main--bacom--adobecom.hlx.page/?milolibs=awesome--milo--forkedowner', 'https://awesome--milo--forkedowner.hlx.live/libs'],
+    ['https://main--bacom--adobecom.hlx.live/', 'https://main--milo--adobecom.hlx.live/libs'],
+    ['https://main--bacom--adobecom.hlx.live/?milolibs=foo', 'https://foo--milo--adobecom.hlx.live/libs'],
+    ['https://main--bacom--adobecom.hlx.live/?milolibs=local', 'http://localhost:6456/libs'],
+    ['https://main--bacom--adobecom.hlx.live/?milolibs=awesome--milo--forkedowner', 'https://awesome--milo--forkedowner.hlx.live/libs'],
+    ['http://localhost:3000', 'https://main--milo--adobecom.hlx.live/libs'],
+    ['http://localhost:3000?milolibs=foo', 'https://foo--milo--adobecom.hlx.live/libs'],
+    ['http://localhost:3000?milolibs=local', 'http://localhost:6456/libs'],
+    ['http://localhost:3000?milolibs=awesome--milo--forkedowner', 'https://awesome--milo--forkedowner.hlx.live/libs'],
```

Comment:
You could probably cut down the number of test cases here. We know that the code will proceed to set the `branch` variable based on the hostname containing `.hlx`, `local`, or `.stage.`. You could check each of those once, and then choose one to check that the branch functionality is working as expected.

Reply:
I wouldn't want to base the test on how the code is written and I want to make sure all of these cases are covered. This test is not fetching resources or doing anything expensive, we could add many more test cases if we so wanted and probably should for lower environments.